www/js/view_single.js

```diff
@@ -20,6 +20,16 @@ var myDropzone = new Dropzone("#form-upload", {
     acceptedFiles: 'image/*'
 });
 
+myDropzone.on("complete", function() {
+  // If all files have been uploaded
+  if (this.getQueuedFiles().length == 0 && this.getUploadingFiles().length == 0) {
```

Comment:
Thanks for adding it!
Also I'm a little confused by the `_this` variable here. It is called from the same function scope, so I don't see its importance, am I missing something?

Reply:
😳 no idea. I added this bit because the uploader was generating duplicates if you tried generating a batch, and then attempted something else within the same session. I googled "dropzone remove processed files from queue" and found this gem. I added it without mentally processing, just testing it.